packages/loader/container-loader/src/test/containerContext.spec.ts

```diff
@@ -114,19 +121,27 @@ describe("ContainerContext Tests", () => {
         mockCodeLoader.verify();
     });
 
+    // open an issue to add string typing
```

Comment:
I think this was "strict" typing from our conversation last week, please do open an issue.

```suggestion
```